hotdoc/core/extension.py

```diff
@@ -482,7 +498,7 @@ def __find_package_root(self):
         if self.__package_root:
             return
 
-        commonprefix = os.path.commonprefix(list(self._get_all_sources()))
+        commonprefix = os.path.commonprefix(list(self._get_all_sources()) + list(self.source_roots))
```

Comment:
Do we still need `__package_root` at all if source_roots have been explicitly specified?

Reply:
Yes, everything is still relative to it internally, user doesn't have to know.